### PR TITLE
Suggestion to show how to uninstall a package

### PR DESCRIPTION
**Pull request by antitree**
I just had an issue installing anti-evil-maid and couldn't find documentation on how to remove it. Looking at the source I think this is the expected way (which worked for me).

### CODE REVIEW

**Comment by axon-qubes** [comment]
@marmarek

File: common-tasks/software-update-dom0.md [hunk 1]
```diff
@@ -66,6 +66,14 @@ Of course, command line tools are still available for accomplishing various upda
     sudo yum downgrade package-version
     ~~~
 
+### How to uninstall a package
+
+If you've installed a package such as anti-evil-maid, you can remove it with the following command:
+
+    ~~~
+    sudo qubes-dom0-update action=remove anti-evil-maid
```
Comment (marmarek) [inline common-tasks/software-update-dom0.md]: For removing better use `yum` directly:
`sudo yum remove anti-evil-maid`

**Comment by axon-qubes** [comment]
@marmarek, should I accept the PR, then make just that change, or does something else need to be changed?

**Comment by marmarek** [comment]
No, that's all.

**Pull request merged**